### PR TITLE
Windows CI build

### PR DESCRIPTION
**Pull request by dominicletz**
As per discussion in #3027 @dgud @garazdawi this is adding a windows CI build based on Github Windows image + WSL.

This build skips building the docs intentionally but produces a Windows installer with all apps (including wxWidgets 3.1.4 and WebView2).

It's not signing any of the executables, but it's definitely good enough to see whether everything still compiles and for quick downloading and testing.

Cheers!

### CODE REVIEW

**Comment by garazdawi** [comment]
Nice!

**Comment by garazdawi** [comment]
Would it be worth adding the smoke tests as well? Just to test that the installer actually works?

From my perspective this is great, just waiting to see if @dgud has any opinions as well.

**Comment by dgud** [comment]
Testing the installer requires admin rights, I don't know if that is available.
And I don't think it is necessary, we have had an offline discussion about the extra windows dll,
but that is more related to wx then this PR.

File: .github/workflows/main.yaml [hunk 1]
```diff
@@ -38,6 +38,78 @@ jobs:
...
+        with:
+          name: otp_git_archive
+
+      - name: Compile Erlang
+        run: |
+          tar -xzf ./otp_src.tar.gz
+          cd otp
+          export ERL_TOP=`pwd`
+          eval `./otp_build env_win32 x64`
+          ./otp_build autoconf
```
Comment (garazdawi) [inline .github/workflows/main.yaml:99]: The otp_src.tar.gz should already be autoconf:ed, so no need to do that.

Comment (dominicletz) [reply .github/workflows/main.yaml]: Thanks! I removed that call

**Comment by dominicletz** [comment]
> Testing the installer requires admin rights, I don't know if that is available.

Admin rights are available, but the installer requires UI interactions I think, e.g. clicking "next" at least. So I don't know how to do that.

But running some kind of smoke tests would be nice. Is there something already available for windows?

**Comment by garazdawi** [comment]
> But running some kind of smoke tests would be nice. Is there something already available for windows?

Could you not run the same smoke test as for Linux? Though being able to run the erlang compiler is a very good smoke test just that, so maybe it is not needed...

**Comment by dgud** [comment]
Use "./Install.exe /S" I think to install without the gui intervention.

**Comment by garazdawi** [comment]
I will include this as it is a great improvement. We can add better functionality of not of this later on if we want to.

Thanks for the PR!

**Pull request merged**